Portfolio/index.html

```diff
@@ -1,177 +1,207 @@
+console.log('Hello World');
+
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <section id="Home"></section>
+    <h1>Doe.</h1>
+    <div class="navbar">
+        <ul>
+            <li>ABOUT</li>
+            <li>SKILLS</li>

```

Comment:
The indentation of the <h1> tag should be adjusted to be consistent with the surrounding tags.